src/App.tsx

```diff
@@ -1,7 +1,40 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type Props = {};
+type State = {
+  pressedKey: string
+};
+
+export class App extends React.Component<Props, State> {
+  state = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', (event: KeyboardEvent) => {
+      this.setState({ pressedKey: event.key });
+    });
+  }
```

Comment:
```suggestion
  componentDidMount() {
    document.addEventListener('keyup', this.handleKey);
  }
  
  handleKey = (event: KeyboardEvent) => {
      this.setState({ pressedKey: event.key })
   }
```
This way , the code block wouldn't be repeated twice.